.github/workflows/release.yaml

```diff
@@ -406,6 +406,100 @@ jobs:
           asset_name: ${{ steps.create_nintendo_3ds_archive.outputs.archive_file }}
           asset_content_type: ${{ steps.create_nintendo_3ds_archive.outputs.archive_content_type }}
 
+  _3ds-cia:
```

Comment:
Do we need to have a separate build step and archive for this? In my opinion, it could just live in the "normal" 3ds archive. That keeps us from having to build the same 3dsx file twice, keeps the code duplication in the workflow low, and ensures that the 3DSX file and the CIA file are as consistent as possible.

Reply:
Done [4efabe6](https://github.com/angband/angband/pull/5376/commits/4efabe63c42fda1dc5779b6d6d79651a115c5282)